includes/class-llms-order-generator.php

```diff
@@ -0,0 +1,597 @@
+<?php
```

Comment:
The purpose of this class is to handle all of the logic currently in [llms_setup_pending_order()](https://developer.lifterlms.com/reference/functions/llms_setup_pending_order/)

This class is more atomic and testable(tested) than the massive setup function.

Additionally, it provides an option to either "commit" (save user information) or just validate it.